source/blood/src/gamemenu.cpp

```diff
@@ -3005,6 +3008,17 @@ bool CGameMenuItemPassword::Event(CGameMenuEvent &event)
 }
 
 CGameMenuFileSelect::CGameMenuFileSelect(const char* _pzText, int _nFont, int _x, int _y, int _nWidth, const char* _startdir, const char* _pattern, char* _destination)
+{
+    InitObject(_pzText, _nFont, _x, _y, _nWidth, _startdir, _pattern, _destination);
+}
+
+CGameMenuFileSelect::CGameMenuFileSelect(const char* _pzText, int _nFont, int _x, int _y, int _nWidth, const char* _startdir, const char* _pattern, char* _destination, void(*_onFileSelectedEventHandler)() )
```

Comment:
Remove other constructor, and set callback default value to nullptr

Reply:
Pushed new commit for this.